src/index.html

```diff
@@ -4,9 +4,375 @@
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__grade-reviews">
+            Reviews: 5
+          </div>
+        </div>
+  
+        <div class="card__price-container">
+          <p class="card__price-text-paragraf">Price:</p>
+          <p class="card__price-paragraf">$2,199</p>
+        </div>
+  
+        <div class="card__buy" >
+          <a class="card__buy-link">BUY</a>
+        </div>
+  
+      </div>
+
+      <div class="card">
+        <img
+          class="card__img"
+          alt="imac"
+          src="images/imac.jpeg"
+        >
+  
+        <div class="card__link">
+          <a>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</a>
+        </div>
+  
+        <div class="card__code">
+          Product code: 195434
+        </div>
+  
+        <div class="card__grade">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__grade-reviews">
+            Reviews: 5
+          </div>
+        </div>
+  
+        <div class="card__price-container">
+          <p class="card__price-text-paragraf">Price:</p>
+          <p class="card__price-paragraf">$2,199</p>
+        </div>
+  
+        <div class="card__buy" >
+          <a class="card__buy-link">BUY</a>
+        </div>
+      </div>
+
+      <div class="card">
+        <img
+          class="card__img"
+          alt="imac"
+          src="images/imac.jpeg"
+        >
+  
+        <div class="card__link">
+          <a>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</a>
+        </div>
+  
+        <div class="card__code">
+          Product code: 195434
+        </div>
+  
+        <div class="card__grade">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__grade-reviews">
+            Reviews: 5
+          </div>
+        </div>
+  
+        <div class="card__price-container">
+          <p class="card__price-text-paragraf">Price:</p>
+          <p class="card__price-paragraf">$2,199</p>
+        </div>
+  
+        <div class="card__buy" >
+          <a class="card__buy-link">BUY</a>
+        </div>
+  
+      </div>
+
+      <div class="card">
+        <img
+          class="card__img"
+          alt="imac"
+          src="images/imac.jpeg"
+        >
+  
+        <div class="card__link">
+          <a>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</a>
+        </div>
```

Comment:
use semantic tags instead of div
this title isn't a link
```suggestion
          <h2>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
```